src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -32,29 +32,58 @@ public class TestJsonCommand : PSCmdlet
         /// then validates the JSON against the schema. Before testing the JSON string,
         /// the cmdlet parses the schema doing implicitly check the schema too.
         /// </summary>
-        [Parameter(Position = 1)]
+        [Parameter(Position = 1, ParameterSetName = TestJsonCommand.SchemaStringParameterSet)]
         [ValidateNotNullOrEmpty()]
         public string Schema { get; set; }
 
+        /// <summary>
+        /// A path to the file containg schema to validate the JSON against.
+        /// This is optional parameter.
+        /// If the parameter is absent the cmdlet only attempts to parse the JSON string.
+        /// If the parameter present the cmdlet attempts to parse the JSON string and
+        /// then validates the JSON against the schema. Before testing the JSON string,
+        /// the cmdlet parses the schema doing implicitly check the schema too.
+        /// </summary>
+        [Parameter(Position = 1, ParameterSetName = TestJsonCommand.SchemaPathParameterSet)]
+        [ValidateNotNullOrEmpty()]
+        public string SchemaPath { get; set; }
```

Comment:
Maybe `SchemaLiteralPath`?
/cc @mklement0

Reply:
Could be ¯\_(ツ)_/¯

Reply:
@iSazonov, given that there is no need for contrast with wildcard-based path handling here, I don't think we need the word _literal_ here, but perhaps `-SchemaFile` is better, given that we're unequivocally dealing with a _file_ path: this would follow the example of `-InFile` and `-OutFile` for `Invoke-WebRequest` / `Invoke-RestMethod`.

 `-SchemaFilePath` is also an option (following the model of `Start-Process`), but I personally don't think that's necessary.

Additionally, to better contrast `-SchemaFile` with `-Schema`, perhaps alias `-SchemaText` could be added.

Reply:
`-SchemaFile` sounds good to me.

Reply:
I've renamed `-SchemaPath` to `-SchemaFile`. Do you want me add `-SchemaText` alias?